src/platforms/android/configuration/integrations/apollo3.mdx

```diff
@@ -0,0 +1,118 @@
+---
+title: Apollo3
+caseStyle: camelCase
+supportLevel: production
+sdk: sentry.java.apollo-3
+description: "Learn more about the Sentry Apollo3 integration for the Android SDK."
+categories:
+  - mobile
+---
+
+<Note>
+
+Capturing transactions requires that you first <PlatformLink to="/performance/">set up performance monitoring</PlatformLink> if you haven't already.
+
+</Note>
+
+Sentry Apollo3 integration provides the `SentryApollo3Interceptor` and the `SentryApollo3HttpInterceptor`, which create a span for each outgoing HTTP request executed with an [Apollo Kotlin](https://github.com/apollographql/apollo-kotlin) GraphQL client. For easier usage, the integration also provides extension functions on the `ApolloClient.Builder`.
+
+## Install
+
+```groovy {tabTitle:Gradle}
+implementation 'io.sentry:sentry-apollo-3:{{ packages.version('sentry.java.apollo-3', '6.1.4') }}'
+```
+
+For other dependency managers, see the [central Maven repository](https://search.maven.org/artifact/io.sentry/sentry-apollo-3).
+
+## Configure With Extension
+
+Add `Interceptors` to `ApolloClient.Builder` using `ApolloBuilderExtensions`:
+
+```java
+import com.apollographql.apollo.ApolloClient;
+import io.sentry.apollo3.ApolloBuilderExtensionsKt;
+
+ApolloClient apollo = ApolloBuilderExtensionsKt.sentryTracing(new ApolloClient.Builder())
+    .serverUrl("https://your-api-host/")
+    .sentryTracing()
```

Comment:
Is this redundant? It isn't included in the example further down under `## Modify or Drop Spans`